components/icons/_icons.scss

```diff
@@ -0,0 +1,5 @@
+// Examples for the pattern library
```

Comment:
I'm not sure but would it be of use to include a comment line here about where where the mixins are called from?